openslides/global_settings.py

```diff
@@ -81,11 +81,12 @@
 
 STATIC_URL = '/static/'
 
+OS_STATICFILES_DIR = os.path.join(MODULE_DIR, 'static')
```

Comment:
Is it possible to remove this settings variable and use just the first entry of `STATICFILES_DIRS` in the code above? This might be less complex.

Reply:
@FinnStutzenstein ?